bin/update-extension-json.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * External dependencies
+ */
+const { getPluginsList, getThemesList } = require( 'wporg-api-client' );
+const axios = require( 'axios' );
+
+/**
+ * Internal dependencies
+ */
+const filesystem = require( './file-system' );
+
+class UpdateExtensionJson {
+	/**
+	 * Construct method.
+	 */
+	constructor() {
+		( async () => {
+			this.plugins = [];
+			this.themes = [];
+
+			await this.fetchData();
+			await this.storeData();
+		} )();
+	}
+
+	/**
+	 * Fetch AMP compatible plugins and themes.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async fetchData() {
+		let totalPage = 1;
+		const pluginTerm = 552;
+		const themeTerm = 245;
+		const url = 'https://amp-wp.org/wp-json/wp/v2/ecosystem';
+
+		const queryParams = {
+			ecosystem_types: [ themeTerm, pluginTerm ],
+			per_page: 20,
```

Comment:
Might as well increase this to 100.
```suggestion
			per_page: 100,
```